src/openforms/utils/tests/test_templatetags.py

```diff
@@ -27,3 +29,35 @@ def test_get_value_with_non_dictionaries(self):
         )
 
         self.assertEqual("", result)
+
+
+@override_settings(SHOW_ENVIRONMENT=True)
```

Comment:
I guess this is done on purpose in order to be more explicit right?I mean its by default True that's why I am asking

Reply:
yeah I want this test to still pass if the defaults in the settings change :)